dist/tokens/evo-dark.css

```diff
@@ -44,5 +44,27 @@
         --color-state-accent-active: #71a6ff;
         --color-state-attention-hover: #ff6988;
         --color-state-attention-active: #ff6383;
+        --color-data-viz-grid: var(--color-neutral-3);
+        --color-data-viz-labels: var(--color-neutral-4);
+        --color-data-viz-legend: var(--color-neutral-2);
+        --color-data-viz-legend-inactive: var(--color-neutral-4);
+        --color-data-viz-legend-hover: var(--color-neutral-0);
+        --color-data-viz-line-chart-primary: var(--color-blue-3);
+        --color-data-viz-line-chart-secondary: var(--color-blue-1);
+        --color-data-viz-line-chart-tertiary: var(--color-teal-3);
+        --color-data-viz-line-chart-queternary: var(--color-magenta-4);
+        --color-data-viz-line-chart-quinary: var(--color-magenta-2);
+        --color-data-viz-trend-positive: var(--color-green-4);
+        --color-data-viz-trend-negative: var(--color-red-3);
+        --color-data-viz-chart-primary: var(--color-blue-4);
+        --color-data-viz-chart-secondary: var(--color-blue-2);
+        --color-data-viz-chart-tertiary-background: var(--color-blue-6);
+        --color-data-viz-chart-tertiary-stroke: var(--color-blue-3);
+        --color-data-viz-chart-quaternary-background: var(--color-teal-5);
+        --color-data-viz-chart-quaternary-stroke: var(--color-teal-3);
+        --color-data-viz-chart-quinary-background: var(--color-teal-7);
+        --color-data-viz-chart-quinary-stroke: var(--color-teal-3);
+        --color-data-viz-tooltip-shadow-primary: #ffffff26;
+        --color-data-viz-tooltip-shadow-secondary: #ffffff2b;
```

Comment:
We should remove these. They should not be needed. (See comment below on the drop shadow for tooltip)